internal/provisioner/plain/manifests/07_webhook.yaml

```diff
@@ -0,0 +1,70 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: rukpak-webhook
+  namespace: rukpak-system
+spec:
+  ports:
+    - port: 443
+      protocol: TCP
+      targetPort: 9443
+  selector:
+    app: plain-provisioner
+
+---
+
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingWebhookConfiguration
+metadata:
+  name: rukpak-webhook
+  annotations:
+    cert-manager.io/inject-ca-from: rukpak-system/rukpak-webhook-certificate
+webhooks:
+- name: core.rukpak.io
```

Comment:
Does this name matter? Should it have `bundle` in the name somewhere?

Reply:
This name probably doesn't matter.  It can serve multiple resources in the same group. So it may be better without the bundle in it.

Reply:
Got it, makes sense.

Reply:
Looking at this closer, it seems like that while you can have multiple resources in the same group, there's only one client config and therefore only one path for the webhook server URL.

It seems like we'll want at least separation of webhook handlers by object kind, in which case it seems like the name _should_ include "bundle" somehow?

Reply:
It must be in domain name format but other than that it doesn't matter.  So I can put `bundle-rukpak-webhook.rukpak-system.svc`